openquake/calculators/extract.py

```diff
@@ -176,3 +177,22 @@ def extract_hazard(dstore, what):
         if oq.poes and oq.hazard_maps:
             hmaps = calc.make_hmap(hcurves, oq.imtls, oq.poes)
             yield 'hmaps-' + kind, convert_to_array(hmaps, N, pdic)
+
+
+@extract.add('losses_by_asset')
+def extract_losses_by_asset(dstore, what):
+    """
+    Extracts losses_by_asset given a `what` string like
+    "rlz-0/structural", "rlz-1/contents_ins", "mean/structural" etc
+    """
+    rlz_or_stat, lt = what.split('/')
+    r = int(rlz_or_stat[4:]) if rlz_or_stat.startswith('rlz-') else None
+    lti = dstore['oqparam'].lti
+    if 'losses_by_asset' in dstore:  # scenario_risk
+        return dstore['losses_by_asset'][:, r, lti[lt]]['mean']
+    elif 'avg_losses-rlzs' in dstore:  # event_based_risk
+        if r is None:
+            assert rlz_or_stat == 'mean', rlz_or_stat
```

Comment:
Please add a comment about the fact that only the mean can be extracted, and no other stat (e.g. quantiles)